common/src/main/java/com/wynntils/features/internal/LootrunFeature.java

```diff
@@ -16,11 +18,10 @@
 import net.minecraft.world.level.block.state.BlockState;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
+@EventListener
+@StartEnabled(false)
```

Comment:
If something should not be enabled on start, maybe remove the need for annotating it? (or the other way around, but I could argue to have a default case, where all features are either enabled or disabled on start by default)

Reply:
I feel like explicitly enabling on start is better though.

Reply:
As long as we have just two internal features, it's kind of pointless to argue which should be default. :-) Let's wait and see if either of the two types come to dominate. I think the default for internal features should be not enabled, and that persistent core features should have `@AlwaysEnabled`. But I don't have strong opinions about this.

Reply:
honestly, i would expect most internal features to be persistent core features like that. the way we enable & disable the lootrun feature is kind of a weird solution for the implementation; maybe we should change it so the feature itself is always enabled, ,and there's a separate flag for whether lootrun logic should be running?

Reply:
I always wanted to disable the feature just so it is unregistered from the event bus. Event bus is pretty slow just for having it return instantly. If we add something that can register/unregister the feature from the event bus (maybe an interface?), then we don't need to actually disable the feature (as I think it's still feels weird to disable internal features imo).

Reply:
Imo event bus registration should be done through feature enabling/disabling, personally think lootrun enabling/disabling could be done via conditions, though the conditon system is clunky